src/generated/resources/data/forge/tags/worldgen/biome/is_plains.json

```diff
@@ -0,0 +1,10 @@
+{
+  "replace": false,
+  "values": [
+    "minecraft:plains",
+    "minecraft:savanna",
+    "minecraft:savanna_plateau",
```

Comment:
I would say remove this from plains tag as plains a low lying flat land. Seems odd to have a plateau in a plains tag

Reply:
also, maybe savanna should also be removed alongside savanna plateau. We already have a savanna tag for targeting savannas. This way people can target plains-like biomes better